packages/utils/web-multi-logger/src/Logger.ts

```diff
@@ -49,16 +49,16 @@ export default class Logger {
    *
    * @returns the port number
    */
-  start(): Promise<number> {
-    return new Promise<number>((resolve) => {
-      this.server = this.app.listen(this.port, () => resolve(this.port));
-    });
+  async startPort(loggerPort: number): Promise<number> {
+    loggerPort = await findAvailablePort(loggerPort);
+    this.server = this.app.listen(loggerPort);
+    return loggerPort;
   }
 
   /**
    * Stop the HTTP server hosting the web UI
    */
-  stop(): void {
+  stopPort(): void {
```

Comment:
I would avoid changing the name of the API if not necessary

```suggestion
  stop(): void {
```